Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -285,9 +285,11 @@ void HexEditorWidget::update_inspector_values(size_t position)
 
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::SignedByte, String::number(static_cast<i8>(unsigned_byte_value)));
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UnsignedByte, String::number(unsigned_byte_value));
+        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, String::repeated(static_cast<char>(unsigned_byte_value), 1));
```

Comment:
That looks pretty confusing, let's just not be clever `String::formatted("{:c}", static_cast<char>(unsigned_byte_value))`

Reply:
Sure, I just couldn't find a quick char->string conversion thing